test/regression/cases/docker_containers_cpu/datadog-agent/checks.d/fake_check.py

```diff
@@ -0,0 +1,12 @@
+import datadog_agent
+from datadog_checks.base import AgentCheck
+
+
+class FakeCheck(AgentCheck):
```

Comment:
Small suggestion to call this `SimpleCheck` or `BasicCheck`. There's nothing "fake" about this, it runs real python code that emits real metrics.

Reply:
Fair point!
`fake_check` renamed `simple_check` in https://github.com/DataDog/datadog-agent/pull/33620/commits/2e40f0a6961d466316c43ddb82f744aabe763cb8.